src/apps/vpn/appfeaturelistcallback.h

```diff
@@ -129,6 +129,16 @@ bool FeatureCallback_splitTunnel() {
     if (VersionUtils::compareVersions(shellVersion, "3.34") < 0) {
       return false;
     }
+  } else if (desktop.contains("KDE")) {
+    QString kdeVersion = LinuxDependencies::kdeFrameworkVersion();
+    if (kdeVersion.isNull()) {
+      return false;
+    }
+    /* The metadata we need (SourcePath) is only added since kio v5.75
+     */
+    if (VersionApi::compareVersions(kdeVersion, "5.75") < 0) {
```

Comment:
`VersionApi::compareVersion()` got moved to `VersionUtils::compareVersions()` during our modularization refactoring and this no longer builds correctly. I will push a fix.